bar-trender-front-end/src/components/ApiFormSubmit.js

```diff
@@ -1,19 +1,20 @@
 import React, { Component } from "react";
+
 import { Col, Button, Form, FormGroup, Label, Input } from "reactstrap";
+import { Link } from "react-router-dom"
+import * as uuid from 'uuid';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7157793)
```suggestion
import * as uuid from "uuid";
```